A critical error has ocurred while running the game!Error type: <class 'AttributeError'>
What happened: 'tuple' object has no attribute 'event'
Where it happened:  file 'd:\Documentos\Programas\C\nundertale\main.py', at line 33Why it happned: Bad coding xD

Full error traceback:
 Traceback (most recent call last):
  File "d:\Documentos\Programas\C\nundertale\main.py", line 33, in main
    controller.main_loop()
  File "d:\Documentos\Programas\C\nundertale\controller\GameController.py", line 30, in main_loop
    for event in self.view.game.event.get():
AttributeError: 'tuple' object has no attribute 'event'
